backend/app/controllers/menu-item.controller.js

```diff
@@ -0,0 +1,59 @@
+const config = require("../config/auth.config");
+const db = require("../models");
+const MenuItem = db.menuitem;
+const Category = require("../models/category.model");
+
+exports.addMenuItem = (req, res) => {
+  const menuitem = new MenuItem({
+    meal: req.body.meal,
+    category: req.body.category,
+  });
+
+  menuitem.save((err, menuitem) => {
+    if(err) {
+      res.status(500).send({ message: err });
+      return;
+    }
+
+    if (req.body.categories) {
+      Category.find(
+        {
+          name: { $in: req.body.categories },
```

Comment:
Shouldn't `req.body.category` be the id of the category? If that's the case, shouldn't we be finding by id here?